### ISSUE TITLE
Release v1.46.4

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#4797]
:robot: A dedicated issue to run tests and deploy release [v1.46.4](https://github.com/deckhouse/deckhouse/milestone/184).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.46.4 | test-v1.46.4*` or a branch `main | release-1.46`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `docker | containerd`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.22 | 1.23 | 1.24 | 1.25 | 1.26 | 1.27`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.46.4 | test-v1.46.4*` or a branch `main | release-1.46`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.23'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.20
/e2e/use/k8s/1.23

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/cri/containerd
/e2e/use/k8s/1.23

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.46
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [8bee2d](https://github.com/deckhouse/deckhouse/commit/8bee2d28b7c39832bfb32f81376363b24faa6216) in branch **main**:
```
[e2e] add third static node and update bootstrap script (#4784)

Signed-off-by: Mikhail Scherba <mikhail.scherba@flant.com>
Co-authored-by: Mikhail Scherba <mikhail.scherba@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [f25b85](https://github.com/deckhouse/deckhouse/commit/f25b85c26f6eb06f741baba6e32d10402e6e9d52) in branch **main**:
```
Re-generate changelog v1.46.3 (#4789)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c41e2f](https://github.com/deckhouse/deckhouse/commit/c41e2f7956a0b05f542b4825217294718746d9a2) in branch **main**:
```
[cni-cilium] Added severity_levels for alerts (#4779)


Signed-off-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [173fe2](https://github.com/deckhouse/deckhouse/commit/173fe295c061d51171ea49f3af2c14256203bf00) in branch **main**:
```
[tools] Set seccomp to unconfined in d8-push and d8-pull scripts

Signed-off-by: alexey.makhonin <alexey.makhonin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [1e1b64](https://github.com/deckhouse/deckhouse/commit/1e1b645ba70be5374a8e18986039e6bdbef5bb16) in branch **main**:
```
[ingress-nginx] Fix kube-rbac-proxy resources indentation (#4738)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [95c895](https://github.com/deckhouse/deckhouse/commit/95c895bad11625a75307ce193e1f80b7f0f34a04) in branch **main**:
```
[linstor] fix kernel-headers cleanup script (#4726)

Signed-off-by: Andrei Kvapil <kvapss@gmail.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [11adba](https://github.com/deckhouse/deckhouse/commit/11adba41922175cc35dd114185f00e8ef13b86fa) in branch **main**:
```
[docs] Add the scheme of using the releaseChannel parameter (#4824)

* [docs] Add the scheme of using the releaseChannel parameter during Deckhouse installation and operation

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b35b27](https://github.com/deckhouse/deckhouse/commit/b35b27831b40caeb51679d26235441da1b50021b) in branch **main**:
```
[linstor] added params for selinux enforced mode support (#4652)

Signed-off-by: v.oleynikov <vasily.oleynikov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [672586](https://github.com/deckhouse/deckhouse/commit/672586fd26a3558436ff20e7490bba0c5d55529f) in branch **main**:
```
[prometheus] Fix d8_prometheus_fs metrics (#4805)

Signed-off-by: Vitaliy Snurnitsin <vitaliy.snurnitsin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ca7352](https://github.com/deckhouse/deckhouse/commit/ca73524eb32683f72476c7e92137fa93d9d9f8cc) in branch **main**:
```
[e2e] update air-gapping procedures (#4804)

Signed-off-by: Mikhail Scherba <mikhail.scherba@flant.com>
Co-authored-by: Mikhail Scherba <mikhail.scherba@flant.com>
```
